M2/Macaulay2/packages/AbstractSimplicialComplexes.m2

```diff
@@ -71,236 +76,194 @@ AbstractSimplicialComplex.GlobalAssignHook = globalAssignFunction
 AbstractSimplicialComplex.GlobalReleaseHook = globalReleaseFunction
 describe AbstractSimplicialComplex := K -> net expression K
 
-
 new AbstractSimplicialComplex := AbstractSimplicialComplex =>(cl) -> (
-    K := newClass(AbstractSimplicialComplex, new HashTable); -- sigh
-    K)
+     K := newClass(AbstractSimplicialComplex, new HashTable); 
+     K)
 
 spots AbstractSimplicialComplex := List => (
-  K -> sort select(keys K, i -> class i === ZZ))
+     K -> sort select(keys K, i -> class i === ZZ))
 
--- return the p-faces of a simplicial complex
+-- Return the p-faces of a simplicial complex.
 
 AbstractSimplicialComplex _ ZZ := AbstractSimplicialComplex => (K,p) -> (
-  if K#?p then K#p 
-  )
+     if K#?p then K#p
+     else {})
```

Comment:
This can be simplified using the "null coalescing operator" `??`, i.e., `K#p ?? {}`.